radix-engine/src/blueprints/resource/non_fungible/non_fungible_vault.rs

```diff
@@ -256,15 +260,12 @@ impl NonFungibleVault {
     where
         Y: ClientApi<RuntimeError>,
     {
-        let handle = api.actor_lock_field(
-            NonFungibleVaultOffset::LiquidNonFungible.into(),
-            LockFlags::read_only(),
+        let items: Vec<NonFungibleLocalId> = api.actor_index_scan_typed(
```

Comment:
Just to check - am I right in thinking we're going to change this API at some point to only return the first N for some N?